docs/release-procedure.md

```diff
@@ -0,0 +1,209 @@
+
+## Release Tool Usage Examples
+
+Once you have your credentials in files, the kubevirt release tool handles
+all the rest. All you need to do is provide your credentials and tell the tool
+what release you want to make.
+
+Place the paths to your credential files in the following environment variables.
+
+```
+export GPG_PRIVATE_KEY_FILE="${HOME}/gpg-private"
+export GPG_PASSPHRASE_FILE="${HOME}/gpg-passphrase"
+export GITHUB_API_TOKEN_FILE="${HOME}/github-api-token"
+```
+
+Now you can use the release tool to do whatever you'd like. Note that you can
+use the ```--dry-run=true``` argument to test a change before executing it.
+
+**Example: creating a new release branch with the initial release candidate v0.31.0-rc.0**
+```
+hack/release.sh --new-branch release-0.31 --new-release v0.31.0-rc.0 --dry-run=false
+```
+
+**Example: Creating a new rc v0.31.0-rc.0**
+```
+hack/release.sh --new-release v0.31.0-rc.0 --dry-run=false
+```
+
+**Example: Promoting a release candidate v0.31.0-rc-1 to official v0.30.0 release.**
+```
+hack/release.sh --promote-release-candidate v0.31.0-rc-1 --dry-run=false
+```
+
+**Example: Creating a patch release v0.31.1. The branch will automatically be detected.**
+```
+hack/release.sh --new-release v0.31.1 --dry-run=false
+```
+
+## Creating New Minor Releases
+The release process is mostly automatic and consists of the following steps:
+
+1. Create the branch and initial RC.
+
+   ```hack/release.sh --new-branch $RELEASE_BRANCH --new-tag ${TAG}.rc.0```
+
+2. Wait 5 business days
+
+3. Promote RC to official release if no blockers exist.
+
+   ```hack/release.sh --promote-release-candidate ${TAG}.rc.0```
+
+4. Wait for [travis](https://travis-ci.org/kubevirt/kubevirt/) to finish, and
+   check that the binary artifacts got attached to the release at
+   `https://github.com/kubevirt/kubevirt/releases/tag/${TAG}`
+   and that the containers were correctly tagged and pushed to
+   <https://hub.docker.com/r/kubevirt/>
+
+5. If release looks correct, click "edit" on the release in the github UI
+   and uncheck the "This is a pre-release" box. This will make the release
+   official
+
+6. Sent a friendly announcement email to <kubevirt-dev@googlegroups.com> using
+   the release notes already present on the release's description in github.
+
+## Creating New Patch Releases
+
+Releases on the stable branch only increment the patch level.
+The release itself is only a git signed tag as it's used for minor releases as well.
+
+1. Create the patch release. Note that the branch is automatically detected.
+
+   ```hack/release.sh --new-tag ${TAG}```
+
+2. Wait for [travis](https://travis-ci.org/kubevirt/kubevirt/) to finish, and
+   check that the binary artifacts got attached to the release at
+   `https://github.com/kubevirt/kubevirt/releases/tag/$TAG`
+   and that the containers were correctly tagged and pushed to
+   <https://hub.docker.com/r/kubevirt/>
```

Comment:
Maybe quay.io?

Reply:
yes, it's `https://quay.io/organization/kubevirt` now i believe

Reply:
fixed